ast/builtins.go

```diff
@@ -588,6 +589,18 @@ var RegexMatch = &Builtin{
 	),
 }
 
+// RegexSplit splits the input string by the occurences of the given pattern.
+var RegexSplit = &Builtin{
+	Name: "re_split",
```

Comment:
Let's namespace this under `regex`, e.g,. `regex.split`.

Reply:
Done